.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,7 +2,7 @@
 name: Feature request
 about: Suggest an idea for implementation
 title: ''
-labels: enhancement / feature request
+labels: enhancement / feature request / Awaiting Triage
```

Comment:
Tags should be separated by commas as per the [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository).  "enhancement / feature request" is a single tag.